bin/transcrypt

```diff
@@ -0,0 +1,1589 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#
+# transcrypt - https://github.com/elasticdog/transcrypt
+#
+# A script to configure transparent encryption of sensitive files stored in
+# a Git repository. It utilizes OpenSSL's symmetric cipher routines and follows
+# the gitattributes(5) man page regarding the use of filters.
+#
```

Comment:
This script is way to large and complex to review for me, and I suppose this is just a local versioning of the transcrypt library in this repo, because Scalingo does not allow installing it as a dependency for some reason. So as we have a local version of it in the repo, why do we still need to install `transcrypt` from homebrew? Can't we always use this one?

Reply:
We need to discuss this part. `transcrypt` is just a bash script but it has dependencies:

> column and hexdump commands (on Ubuntu/Debian install bsdmainutils)
> if using OpenSSL 3+ one of: xxd (on Ubuntu/Debian is included with vim) or printf command (with %b directive) or perl

That's why we need to install it (to be sure the dependencies are installed). But I suppose you're right, I may remove it from the repo, download it on Scalingo when deploying and rely on the user to install it locally.